src/fps/example/Runner.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity 0.8.15;
```

Comment:
Now that we have our new `test/` directory, should these files be moved under there?